src/paho/mqtt/enums.py

```diff
@@ -0,0 +1,88 @@
+import enum
+
+
+# Error values
+class MQTTErrorCode(enum.IntEnum):
+    MQTT_ERR_AGAIN = -1
+    MQTT_ERR_SUCCESS = 0
+    MQTT_ERR_NOMEM = 1
+    MQTT_ERR_PROTOCOL = 2
+    MQTT_ERR_INVAL = 3
+    MQTT_ERR_NO_CONN = 4
+    MQTT_ERR_CONN_REFUSED = 5
+    MQTT_ERR_NOT_FOUND = 6
+    MQTT_ERR_CONN_LOST = 7
+    MQTT_ERR_TLS = 8
+    MQTT_ERR_PAYLOAD_SIZE = 9
+    MQTT_ERR_NOT_SUPPORTED = 10
+    MQTT_ERR_AUTH = 11
+    MQTT_ERR_ACL_DENIED = 12
+    MQTT_ERR_UNKNOWN = 13
+    MQTT_ERR_ERRNO = 14
+    MQTT_ERR_QUEUE_SIZE = 15
+    MQTT_ERR_KEEPALIVE = 16
+
+
+class MQTTProtocolVersion(enum.IntEnum):
+    MQTTv31 = 3
+    MQTTv311 = 4
+    MQTTv5 = 5
+
+
+class MessageType(enum.IntEnum):
+    CONNECT = 0x10
+    CONNACK = 0x20
+    PUBLISH = 0x30
+    PUBACK = 0x40
+    PUBREC = 0x50
+    PUBREL = 0x60
+    PUBCOMP = 0x70
+    SUBSCRIBE = 0x80
+    SUBACK = 0x90
+    UNSUBSCRIBE = 0xA0
+    UNSUBACK = 0xB0
+    PINGREQ = 0xC0
+    PINGRESP = 0xD0
+    DISCONNECT = 0xE0
+    AUTH = 0xF0
+
+class PahoLogLevel(enum.IntEnum):
+    MQTT_LOG_INFO = 0x01
+    MQTT_LOG_NOTICE = 0x02
+    MQTT_LOG_WARNING = 0x04
+    MQTT_LOG_ERR = 0x08
+    MQTT_LOG_DEBUG = 0x10
+
+
+class ConnackCode(enum.IntEnum):
+    CONNACK_ACCEPTED = 0
+    CONNACK_REFUSED_PROTOCOL_VERSION = 1
+    CONNACK_REFUSED_IDENTIFIER_REJECTED = 2
+    CONNACK_REFUSED_SERVER_UNAVAILABLE = 3
+    CONNACK_REFUSED_BAD_USERNAME_PASSWORD = 4
+    CONNACK_REFUSED_NOT_AUTHORIZED = 5
+
+
+class ConnectionState(enum.IntEnum):
+    mqtt_cs_new = 0
+    mqtt_cs_connected = 1
+    mqtt_cs_disconnecting = 2
+    mqtt_cs_connect_async = 3
```

Comment:
These should maybe also be `UPPER_SNAKE_CASE`? The compatibility aliases in `client.py` can of course remain lower-case.